schemas/core/validator-log.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$id": "https://raw.githubusercontent.com/guardrails-ai/interfaces/core-schemas/schemas/core/validator-log.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "ValidatorLog",
+  "type": "object",
+  "properties": {
+    "validatorName": {
+      "type": "string",
+      "description": "The class name of the validator.",
+      "format": "PascalCase"
```

Comment:
I always thought this was called TitleCase

Reply:
technically title case is "Title Case"